tests/test_private_vlan.rb

```diff
@@ -0,0 +1,729 @@
+# NXAPI New test for feature private-vlan
```

Comment:
Again, based on our earlier discussion reminder to validate default states for both `private_vlan_type` and `private_vlan_association`